assets/css/styles.css

```diff
@@ -0,0 +1,387 @@
+a {
+  text-decoration: none;
+}
+
+ul {
+  list-style: none;
+}
+
+.homepage {
+  display: grid;
+  position: relative;
+  width: 414px;
+  height: 5410px;
+  background: #f7f7f9;
+}
+
+header {
+  position: absolute;
+  width: 414px;
+  height: 718px;
+  left: 0;
+  top: 0;
+  background: #fff;
+  border-bottom-left-radius: 70px;
+}
+
+nav {
+  position: absolute;
+  width: 414px;
+  height: 72px;
+  left: 0;
+}
+
+.nav-container {
+  position: absolute;
+  left: 0;
+  right: 0;
+  top: 10px;
+  bottom: 0;
+}
+
+.logo {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  padding: 10px 12px;
+  position: absolute;
+  width: 100px;
+  height: 40px;
+  left: 12px;
+  top: calc(50% - 40px / 2);
+}
+
+.logo img {
+  position: static;
+  width: 76px;
+  height: 20px;
+  left: 12px;
+  top: 10px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 18px;
+  line-height: 20px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  color: #6070ff;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0 0;
+}
+
+.menu-icon {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  right: 24px;
+  top: calc(50% - 40px / 2);
+  border-radius: 8px;
+}
+
+.menu-icon img {
+  position: static;
+  left: 20%;
+  right: 20%;
+  top: 20%;
+  bottom: 20%;
+  flex: none;
+  order: 0;
+  flex-grow: 0;
+  margin: 0 0;
+}
+
+.header-container {
+  position: absolute;
+  width: 412px;
+  height: 587.15px;
+  left: 0;
+  top: 10px;
+}
+
+.rectangle7 {
+  position: absolute;
+  width: 26.46px;
+  height: 4.73px;
+  left: 127.18px;
+  top: 121.59px;
+  background: #6070ff;
+  transform: matrix(0.76, 0.65, 0.65, -0.76, 0, 0);
+}
+
+.ellipse5 {
+  position: absolute;
+  width: 10px;
+  height: 10px;
+  left: 132.98px;
+  top: 155.81px;
+  background: #cbcdff;
+  border-radius: 50%;
+  transform: rotate(52.97deg);
+}
+
+.rectangle5 {
+  position: absolute;
+  width: 68.06px;
+  height: 12.18px;
+  left: 191.82px;
+  top: 670px;
+  background: #ebebff;
+  transform: rotate(23.34deg);
+}
+
+.ellipse4 {
+  position: absolute;
+  width: 20px;
+  height: 20px;
+  left: 237px;
+  top: 670px;
+}
+
+.ellipse3 {
+  position: absolute;
+  width: 90px;
+  height: 105px;
+  left: 326px;
+  top: 152.24px;
+}
+
+.ellipse13 {
+  position: absolute;
+  width: 30.1px;
+  height: 30.1px;
+  left: 332.18px;
+  top: 620px;
+  transform: rotate(69.41deg);
+}
+
+.ellipse12 {
+  position: absolute;
+  width: 24.14px;
+  height: 12.07px;
+  left: 341.76px;
+  top: 620px;
+  transform: rotate(-176.08deg);
+}
+
+.rectangle6 {
+  position: absolute;
+  width: 51.99px;
+  height: 9.3px;
+  left: -34px;
+  top: 100.49px;
+  background: #ebebff;
+  transform: rotate(-46.14deg);
+}
+
+.ellipse11 {
+  position: absolute;
+  width: 38.5px;
+  height: 19.25px;
+  left: -20px;
+  top: 131.47px;
+  transform: rotate(-156.88deg);
+}
+
+.ellipse8 {
```

Comment:
[OPTIONAL]
- Your ``.ellipse8`` and ``.ellipse15`` contains similar parameters you could improve your code using DRY _(Optimized code and less lines)._

Reply:
I will appreciate a hint to apply this to both .ellipse8 and .ellipse15 from a single point. They are both different images. Thank you.

Reply:
You can found more information about that [here](https://vanseodesign.com/css/dry-principles/).
- This is a way to DRY your code:
```
.ellipse8
.ellipse15  {
 position: absolute;
  width: 15px;
  height: 15px;
  background: #fff;
  box-sizing: border-box;
}

.ellipse8 {
  left: 214px;
  top: 110px;
}

.ellipse15  {
  left: 53px;
  top: 680px;
}
 ```
 
 _I hope that this information going to be helpful for you._

Reply:
Thank you